source/Firebase/Functions/Functions.csproj

```
@@ -0,0 +1,86 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+  <PropertyGroup>
+    <TargetFrameworks>Xamarin.iOS10</TargetFrameworks>
+    <IsBindingProject>true</IsBindingProject>
+    <RootNamespace>Firebase.Functions</RootNamespace>
```

Suggestion to change it to CloudFunctions where applicable.

```suggestion
    <RootNamespace>Firebase.CloudFunctions</RootNamespace>
```